runtime/binding-kafka/src/main/java/io/aklivity/zilla/runtime/binding/kafka/internal/stream/KafkaClientProduceFactory.java

```diff
@@ -1654,85 +1634,135 @@ private void doEncodeRecordInit(
                 KafkaAckMode ackMode,
                 KafkaKeyFW key,
                 OctetsFW value,
-                Array32FW<KafkaHeaderFW> headers)
+                Array32FW<KafkaHeaderFW> headers,
+                long producerId,
+                short producerEpoch,
+                int sequence)
```

Comment:
Let's move parameters `producerId`, `producerEpoch` and `sequence` between parameters `timestamp` and `ackMode` for consistent ordering with 
```
        struct KafkaProduceDataEx
        {
            int32 deferred = 0;
            int64 timestamp = 0;
            int64 producerId = -1;
            int16 producerEpoch = -1;
            int32 sequence = -1;
            uint32 crc32c = 0;
            KafkaAckMode ackMode = IN_SYNC_REPLICAS;
            KafkaKey key;
            KafkaHeader[] headers;
        }
```